CRM/Utils/System/Standalone.php

```diff
@@ -313,15 +313,10 @@ public function loadBootStrap($params = [], $loadUser = TRUE, $throwError = TRUE
 
     if (!isset($runOnce)) {
       $runOnce = TRUE;
-      return TRUE;
     }
 
-    $root = rtrim($this->cmsRootPath(), '/' . DIRECTORY_SEPARATOR);
-    if (empty($root) || !is_dir($root) || !chdir($root)) {
```

Comment:
Tried to figure out why the `chdir()` was here in the first place. I'm guessing it's just copied over from Drupal/Backdrop, we don't do it for other CMS and removing it doesn't seem to break anything here.